python/lsst/daf/butler/configs/datastores/fileDatastore.yaml

```diff
@@ -5,6 +5,9 @@ datastore:
     table: file_datastore_records
   create: true
   templates:
-    default: "{run:/}/{datasetType}.{component:?}/{tract:?}/{patch:?}/{label:?}/{band:?}/{subfilter:?}/{physical_filter:?}/{visit:?}/{datasetType}_{component:?}_{tract:?}_{patch:?}_{label:?}_{band:?}_{physical_filter:?}_{visit:?}_{exposure:?}_{detector:?}_{instrument:?}_{skymap:?}_{skypix:?}_{run}"
+    default: "{run:/}/{datasetType}.{component:?}/{tract:?}/{patch:?}/{visit.day_obs:?}/{exposure.day_obs:?}/{band:?}/{subfilter:?}/{physical_filter:?}/{visit:?}/{exposure.obs_id:?}/{datasetType}_{component:?}_{tract:?}_{patch:?}_{band:?}_{physical_filter:?}_{visit:?}_{exposure.obs_id:?}_{detector.full_name:?}_{instrument:?}_{skymap:?}_{skypix:?}_{run}"
+    # For raw-type files do not include band or filter in hierarchy
+    # and remove band from file name
+    physical_filter+detector+exposure: "{run:/}/{datasetType}.{component:?}/{exposure.day_obs}/{exposure.obs_id}/{datasetType}_{component:?}_{instrument:?}_{physical_filter}_{exposure.obs_id}_{detector.full_name}_{run}"
```

Comment:
Seems a little strange that `instrument` moves around.  You don't want it before `band` in the default template, for example?

Reply:
Good point. I've moved instrument to the front in the default to match.